mdx_oembed/endpoints.py

```diff
@@ -6,6 +6,14 @@
     'https?://youtu.be/*',
 ])
 
+SLIDESHARE = oembed.OEmbedEndpoint('http://www.slideshare.net/api/oembed/2', [
+    'http://www.slideshare.net/*/*',
```

Comment:
these URL patterns should probably be merged into one (`http://(*.)?slidesha...`)